lib/stripe/error.ex

```diff
@@ -166,67 +150,35 @@ defmodule Stripe.Error do
     end
   end
 
-  defp code_from_status(status) do
-    case status do
-      400 -> :bad_request
-      401 -> :unauthorized
-      402 -> :request_failed
-      404 -> :not_found
-      409 -> :conflict
-      429 -> :too_many_requests
-      s when s in [500, 502, 503, 504] -> :server_error
-      _ -> :unknown_error
-    end
-  end
```

Comment:
Different function clauses are preferable, and AFAIK, faster than a single function wrapped around a large case statement.